src/duke/TaskList.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+import java.util.ArrayList;
+public class TaskList {
+    private ArrayList<Task> arr;
+    TaskList(ArrayList<Task> arr) {
+        this.arr = arr;
+    }
+
+    /**
+     * Getter fot the arr.
```

Comment:
Is this a typo? Do you mean `Getter for the arr`? In addition, for the JavaDoc description of a method, you may start using a verb. So maybe it'll be better to say something like `Gets the arr.`?